coba/benchmarks/results.py

```diff
@@ -533,6 +535,9 @@ def plot_learners(self,
         if show:
             plt.show()
 
+        if filename:
```

Comment:
or we could also return plt, let me know if you have any preference

Reply:
I like it. Yeah I've just been saving the data to file so I could recreate the plots but this is an obvious use case.